### PR TITLE
Web radio

### PR DESCRIPTION
**Pull request by julizet**
woudl like to add article about webRadio to lab site
happy if someone could quickly review for typos (did it already but feels already like "betriebslind" ;))

### CODE REVIEW

File: source/projects/webRadio/de/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+{% include macro-image-section-markdown.html src="../images/webRadio_network.jpeg" caption="Schnittstellen zwischen den einzelnen Devices (©eigene Darstellung, Technologiestiftung Berlin) "%}
+
+## RasPi mit dem Radio verbinden
+Die meisten von uns verbinden sich tagtäglich via Bluetooth mit anderen Devices – ganz einfach unter Nutzung des Graphic User Interface (GUI) mit Tastatur und Maus. Das ist in der Regel auch schneller, doch in unserem Workshop stellen wir die Verbindung lediglich mit ein paar Terminal-Command her, denn genau hier liegt die **Magie der Automation**. Speichert man die Commands in einem Shell-Skript und bindet dieses kleine Skript in den Autostart des Raspberry's ein, verbindet sich der Raspberry beim nächsten Hochfahren automagically mit dem Bluetooth-Radio. 
+
+Zum Verbinden mit dem Radio sind folgenden Befehle notwendig (wobei die angegebene MAC-Adresse des Radios durch die des eigenen Radios ausgetauscht werden muss):
+```shell
+$ bluetoothctl
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: If you use `$` to mark rhe command you should also use `>` to mark the response from the shell. Also you should mention that it is the command prompt and not part of the command

File: source/projects/webRadio/de/index.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+Wer jetzt neugierig geworden ist und gerne selber aktive werden möchte, findet weitere Instruktionen im ausführlichen [Tutorial auf GitHub](https://github.com/technologiestiftung/werkstatt/blob/master/HowTo_WebRadio.md). Darin wird unter anderem erklärt wie:
+* die IP des RasPi's abfragt werden kann
+* der RasPi über SSH angesteuert werden kann
+* die .desktop-Autostart-Datei angelegt und abgelegt werden sollte
+* der Audio-Ausgang des Raspberry eingerichtet werden kann
+* uvm.
+
+
+Wer zu Hause etwas experimentieren jedoch keinen eigenen RasPi besitz, der kann sich **RasPi's + Zubehör kostenlos via [Hacking Box](https://www.technologiestiftung-berlin.de/hackingbox/) ausleihen.** Darüber hinaus bietet das [CityLAB Berlin](https://www.citylab-berlin.org/) monatlich stattfindende, partizipative Workshops zu Themen wie Internet of Things, Mikrokontroller, Smart Home und Hardware Hacking an.
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: experimentieren **möchte** jedoch

File: source/projects/webRadio/de/index.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+---
+
+*Auch das Ideation & Prototyping Lab möchten im Zeitalter der zunehmenden Automation Schritt halten und hat daher das hauseigene Bluetooth-Radio des [CityLAB Berlin](https://www.citylab-berlin.org/) zum Web Radio transformatiert. Das dabei entstandene Tutorial beschäftigt sich mit dem **Raspberry Pi**, dessen **Autostart-Konfiguration** & **Bluetooth Radios**. Wie das Alles zusammenhängt und wie man sich mit Hilfe eines RasPi's ein Autoplay Web Radio hackt, das soll in diesem Artikel beschrieben werden. Das ausführlich Tutorial findet ihr auf [GitHub](https://github.com/technologiestiftung/werkstatt/blob/master/HowTo_WebRadio.md)*.
+
+## Die Idee
+In Zeiten der zunehmenden Automation wollten wir einmal mehr hinter die Kulissen blicken und ein bluetooth-fähiges Radio mit Hilfe eines Raspberry Pi 3 und einem **Autostart-Programm** zu einem Autoplay Web Radio hacken. Die Kommunikation vom Laptop zum RasPi erfolgt dabei über SSH - vom RasPi zum Radio über Bluetooth. Die gesamte Konfiguration (Bluetooth, VLC-Mediaplayer Command, Delays etc.) wurde in ein **Shell-Skript** geschrieben, welches schließlich mit der Autostart-Datei des Raspberry's verknüpft wurde.
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: I would suggest to add at the fist  occurrence of `Raspberry Pi` the abbreviation `(RasPi)`

File: source/projects/webRadio/de/index.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+## Web Radio per Autostart abspielen
+
+Damit der Aha-Effekt größer und das Ergebnis hörbar ist, haben wir das Shell-Skript um einen weiteren Command ergänzt. Mit dem Command
+```shell
+$ vlc meinePlaylist.m3u
+```
+öffnet der bereits vorinstalliert VLC-Mediaplayer die eigens erstellte Playlist "meinePlaylist". Alle darin befindlichen Dateien wie mp3-Dateien oder weitere Playlists, beispielsweise die Playlist eures Lieblings-Web-Radios, werden mit dem Command nacheinander abgespielt. In die Playlist muss demnach lediglich eine URL zu entsprechenden Web-Radiosender hinterlegt werden.
+
+**Last but not least: das Shell-Skript in den Autostart einbinden.** Um ein Programm in den Autostart des Raspberry's einzubinden gibt es verschiedene Möglichkeiten. In unserem Tutorial haben wir eine **.dektop-Datei** erstellt, um unser Skript mit jedem Hochfahren des RasPi's auszuführen.
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: ~~**.dektop-Datei**~~ **.desktop-Datei**

File: source/projects/webRadio/de/index.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+$ sudo geany /etc/xdg/autostart/NameDerDatei.desktop&
+```
+Nice To Know: durch das kaufmännische "Und" am Ende des Befehls, könnt ihr das Terminal weiterhin bedienen, obwohl der Texteditor geöffnet ist.
+Die .desktop-Dateien haben eine vorgeschriebene schematische Struktur, die es zu berücksichtigen gilt.
+```
+[Desktop Entry]
+Type=Application
+Name=irgendeinName
+Terminal=false
+Exec=sh /usr/bin/meinSkript.sh
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: Better to use `/bin/sh` then only `sh` you never now when the file is loaded and if `bin` is already in `$PATH`

File: source/projects/webRadio/de/index.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+```
+Nach dem Attribut "Exec=..." wird der beim Autostart auszuführende Befehl angegeben. Da wir für die Verbindung via Bluetooth und das Abspielen des Web-Radiosenders mehrere Befehle benötigen, verlinken wir an dieser Stelle also zu unserem bereits erstellten Shell Skript "meinSkript.sh".
+
+## Ausblick
+
+Wer jetzt neugierig geworden ist und gerne selber aktive werden möchte, findet weitere Instruktionen im ausführlichen [Tutorial auf GitHub](https://github.com/technologiestiftung/werkstatt/blob/master/HowTo_WebRadio.md). Darin wird unter anderem erklärt wie:
+* die IP des RasPi's abfragt werden kann
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: Add a `.` to the end of each bullet list item

File: source/projects/webRadio/de/index.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+* der RasPi über SSH angesteuert werden kann
+* die .desktop-Autostart-Datei angelegt und abgelegt werden sollte
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: Name of the file changed to `.desktop-Autostart-Datei` from `.dektop-Datei`

File: source/projects/webRadio/de/index.md (hunk 6 above)
Comment (julizet) [reply source/projects/webRadio/de/index.md]: added `.`

File: source/projects/webRadio/de/index.md (hunk 5 above)
Comment (julizet) [reply source/projects/webRadio/de/index.md]: nice to now --> changed to `/bin/sh`

File: source/projects/webRadio/de/index.md (hunk 3 above)
Comment (julizet) [reply source/projects/webRadio/de/index.md]: added abbreviation within the teaser

File: source/projects/webRadio/de/index.md (hunk 1 above)
Comment (julizet) [reply source/projects/webRadio/de/index.md]: wasn't quite sure how to signal the response. However, I used `>` now and stated a cue relating to the comments

File: source/projects/webRadio/de/index.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+## RasPi mit dem Radio verbinden
+Die meisten von uns verbinden sich tagtäglich via Bluetooth mit anderen Devices – ganz einfach unter Nutzung des Graphic User Interface (GUI) mit Tastatur und Maus. Das ist in der Regel auch schneller, doch in unserem Workshop stellen wir die Verbindung lediglich mit ein paar Terminal-Command her, denn genau hier liegt die **Magie der Automation**. Speichert man die Commands in einem Shell-Skript und bindet dieses kleine Skript in den Autostart des Raspberry's ein, verbindet sich der Raspberry beim nächsten Hochfahren automagically mit dem Bluetooth-Radio. 
+
+Zum Verbinden mit dem Radio sind folgenden Befehle notwendig (wobei die angegebene MAC-Adresse des Radios durch die des eigenen Radios ausgetauscht werden muss):
+```shell
+$ bluetoothctl
+Agent registered
+
+$ [bluetooth]# power on
```
[the remark this answers is not in the document]
Comment (julizet) [reply source/projects/webRadio/de/index.md]: "Agent registered" is actually not a command. It's rather the respond of the terminal. As soon as you start bluetooth config an agent is registered

[the remark this answers is not in the document]
Comment (fabianmoronzirfas) [reply source/projects/webRadio/de/index.md]: Yes. That's why I suggested using the `>` to indicate that

File: source/projects/webRadio/de/index.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+Zum Verbinden mit dem Radio sind folgenden Befehle notwendig, wobei die angegebene MAC-Adresse des Radios durch die des eigenen Radios ausgetauscht werden muss (Kommentare werden nicht im Terminal angezeigt):
+```shell
+$ bluetoothctl
+> Agent registered
+
+$ [bluetooth]# power on
+> changing power on succeeded
+
+$ [bluetooth]# default-agent
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: I would suggest not to add the `# comment` in the examples behind the lines. Add it before or split the code blocks and write the comments in between

File: source/projects/webRadio/de/index.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+> Default agent request successful
+
+$ [bluetooth]# pairable on
+> changing pairable on succeeded
+
+//wichtiger command, falls man sich erneut(!) mit dem gleichen Gerät verbinden möchte
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: `//` is not a shell comment. Use `#`

File: source/projects/webRadio/de/index.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+
+{% include macro-image-section-markdown.html src="../images/webRadio_network.jpeg" caption="Schnittstellen zwischen den einzelnen Devices (©eigene Darstellung, Technologiestiftung Berlin) "%}
+
+## RasPi mit dem Radio verbinden
+Die meisten von uns verbinden sich tagtäglich via Bluetooth mit anderen Devices – ganz einfach unter Nutzung des Graphic User Interface (GUI) mit Tastatur und Maus. Das ist in der Regel auch schneller, doch in unserem Workshop stellen wir die Verbindung lediglich mit ein paar Terminal-Command her, denn genau hier liegt die **Magie der Automation**. Speichert man die Commands in einem Shell-Skript und bindet dieses kleine Skript in den Autostart des Raspberry's ein, verbindet sich der Raspberry beim nächsten Hochfahren automagically mit dem Bluetooth-Radio. 
+
+Zum Verbinden mit dem Radio sind folgenden Befehle notwendig, wobei die angegebene MAC-Adresse des Radios durch die des eigenen Radios ausgetauscht werden muss (Kommentare werden nicht im Terminal angezeigt):
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: Add a line that explains what `#`, `>` and `$` means.

File: source/projects/webRadio/de/index.md (hunk 4 above)
Comment (fabianmoronzirfas) [reply source/projects/webRadio/de/index.md]: It is still `dektop` not `desktop`

File: source/projects/webRadio/de/index.md [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+* die IP des RasPi's abfragt werden kann.
+* der RasPi über SSH angesteuert werden kann.
+* die .desktop-Autostart-Datei angelegt und abgelegt werden sollte.
+* der Audio-Ausgang des Raspberry eingerichtet werden kann.
+* uvm.
+
+
+Wer zu Hause etwas experimentieren möchte, jedoch keinen eigenen RasPi besitz, der kann sich **RasPi's + Zubehör kostenlos via [Hacking Box](https://www.technologiestiftung-berlin.de/hackingbox/) ausleihen.** Darüber hinaus bietet das [CityLAB Berlin](https://www.citylab-berlin.org/) monatlich stattfindende, partizipative Workshops zu Themen wie Internet of Things, Mikrokontroller, Smart Home und Hardware Hacking an. 
+
+Bei Fragen rund um das Thema *RasPi & Web Radio* hilft [Julia Zimmermann](mailto:Zimmermann@technologiestiftung-berlin.de) gerne weiter.
```
Comment (fabianmoronzirfas) [inline source/projects/webRadio/de/index.md]: Mails in Markdown can be written as such `<foo@bah.com>` but you don't have to.

File: source/projects/webRadio/de/index.md (hunk 9 above)
Comment (julizet) [reply source/projects/webRadio/de/index.md]: actually, the `#` comes together with bluetoothctl. It is not meant as a comment. eg. `[bluetoothctl]# ` comes directly out of command line and `power on` is the actual command

File: source/projects/webRadio/de/index.md (hunk 12 above)
Comment (julizet) [reply source/projects/webRadio/de/index.md]: didn't changed it but will keep it in mind for future articles

File: source/projects/webRadio/de/index.md (hunk 4 above)
Comment (julizet) [reply source/projects/webRadio/de/index.md]: finally ;)

**Pull request merged**